api/envoy/extensions/transport_sockets/tls/v3/tls_spiffe_validator_config.proto

```diff
@@ -57,4 +82,10 @@ message SPIFFECertValidatorConfig {
 
   // This field specifies trust domains used for validating incoming X.509-SVID(s).
   repeated TrustDomain trust_domains = 1 [(validate.rules).repeated = {min_items: 1}];
+
+  // This field specifies all trust bundles as a single file. If both
+  // trust_bundles and trust_domains are specified, trust_bundles will
+  // take precedence. Currently assumes file will be a SPIFFE Trust Bundle Map.
+  config.core.v3.DataSource trust_bundles = 2;
```

Comment:
Also please add a comment to tell the end user the dynamic file watching will enabled in this field. Because int most use cases of the DataSource, the file watching is not supported.